src/docs/command-line.dox

```diff
@@ -324,12 +324,13 @@ By default, the \c loadtext command reads from the standard input; the
 
 <hr>
 @section util_printlog wt printlog
-Display the database log.
+Display the database log. By default any operations in the log containing user
+data are redacted.
```

Comment:
Is this right? User data is redacted, but the operations themselves are displayed?

Reply:
No, the entire operation is redacted if it contains user data. I'm not sure how you're reading that the operation is displayed?

Reply:
I anticipated the operation itself would be displayed ("transaction XXX changed a row"), but the data itself (the key/value pair), be redacted. Does that make sense?  If that's not what we did, just ignore me!

Reply:
That is not what I did. Te entire operation is redacted. It seemed to me that knowing the type and order of operations could leak something even if it wasn't explicitly user data. So I redact the whole thing.

Reply:
Yes, I would agree even knowing an operation occurred is a covert channel. Thanks for explaining.